packages/excalidraw/scene/export.ts

```diff
@@ -541,7 +541,10 @@ function* fontFacesIterator(
             return;
           }
 
-          const fontFaceOrder = familyIndex + fontFaceIndex;
+          const fontFaceOrder = Number(
+            familyIndex.toString() + fontFaceIndex.toString(),
+          );
```

Comment:
this is potentially buggy as well as you _could_ have `"1" + "0"` and `"0" + "10"` or combinations, so we'll need to use something like cantor pairing or similar:

`(a: number, b: number) => ((a + b) * (a + b + 1)) / 2 + b`

Reply:
yep, I had there a buffer pair family before, not sure where it disappear, will re-introduce